.gitignore

```diff
@@ -1,6 +1,8 @@
 cmd/movefunds/movefunds
 cmd/sweepaccount/sweepaccount
-dcrwallet
+cmd/repaircfilters/repaircfilters
+cmd/treasurykey/treasurykey
+/dcrwallet
```

Comment:
Can this be broken out into a different PR?

Reply:
Sure thing.  The cmd ignores are not relevant, but git was spamming me because of the rpc/client/dcrwallet paths changed in this pr were matching the dcrwallet glob.

Reply:
Oh. Yeah let's fix that then.  I wasn't sure what the reasoning was here.